Java/LocalDateTime.md

```diff
@@ -19,3 +19,72 @@ Camel’s case in java programming consists of compound words or phrases such th
 |static LocalDateTime of(LocalDate date, LocalTime time)	|It is used to obtain an instance of LocalDateTime from a date and time.|
 |LocalDateTime plusDays(long days)	|It is used to return a copy of this LocalDateTime with the specified number of days added.|
 |boolean equals(Object obj)	|It is used to check if this date-time is equal to another date-time.|
+
```

Comment:
Please go through other review comments for LocalDateTime.
Seperate your content part from the code

Reply:
Done !